nitro-protocol/gas-benchmarks/exportBenchmarkResults.ts

```diff
@@ -1,5 +1,9 @@
 import {writeFileSync} from 'fs';
```

Comment:
What is the difference between `benchmark.test.ts` and `exportBenchmarkResults.ts`?

Reply:
[Explain difference between files](https://github.com/statechannels/go-nitro/pull/831/commits/d3f8627fa5cee6f130860d9542a7b35c7728dfc5)